readme.md

```diff
@@ -638,6 +638,8 @@ Usage:
 </AppContext.Consumer>
 ```
 
+If `exit` is called with a value, `waitUntilExit` will reject with the value.
```

Comment:
This should be enforced to be an error.

Reply:
Should we check that it's an Error instance in the code or just document it as such?

Reply:
I would check it in the code too, as it could catch hard-to-debug mistakes.

Reply:
Updated.